api/MessageEvent.json

```diff
@@ -54,7 +54,7 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/comms.html#the-messageevent-interface:dom-event-constructor",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "16"
```

Comment:
Hmm, how did you determine this? BrowserStack doesn't have Chrome 15. I wanted to check how the test fails in Chrome 15, to confirm it's not actually supported.

Reply:
It does have Chrome 15, actually!  That's why the collector's lowest version to run Selenium on is Chrome 15.

Reply:
Hmm, I guess I was looking on the wrong platforms then. Can you paste the error message from Chrome 15 when trying this constructor?

Reply:
The resulting error is pretty straightforward: `threw TypeError: Illegal constructor`